public/catch-js-errors.js

```diff
@@ -118,6 +121,23 @@
 					savedWindowOnError();
 				}
 			};
+		} else if ( localStorageAssignment === 'analytics' ) {
```

Comment:
Maybe the assignment should be 'ga'? Would we never log to other analytics tools like tracks?